pkg/frontend/mysql_cmd_executor.go

```diff
@@ -2169,7 +2178,6 @@ func (mce *MysqlCmdExecutor) ExecRequest(requestCtx context.Context, req *Reques
 		return resp, nil
 	case COM_QUERY:
 		var query = string(req.GetData().([]byte))
-		mce.addSqlCount(1)
```

Comment:
just keep it , the computation engine may need it. Get response from @nnsgmsone  before removing it.

Reply:
> just keep it , the computation engine may need it. Get response from @nnsgmsone before removing it.

What is this, statement id? The statement id would be useful for solving the Halloween problem, though I'm not sure if tae could use this information. I think the statement id could be used to solve the problem, though tae's solution I don't know.

Reply:
> 

It forms the process.Id. Does process.Id can be replaced by the UUID here?

Reply:
> > 
> 
> It forms the process.Id. Does process.Id can be replaced by the UUID here?

may not suitable, caused by uuid stores in `ComputationWrapper`, not in `MysqlCmdExecutor`